packages/ng/styles/definitions/select/_select-input.scss

```diff
@@ -231,16 +231,39 @@
 
 	// Multiple
 	:host(.mod-multiple) {
-			.lu-select-value {
-				height: auto;
-				align-items: center;
+		.lu-select-value {
+			height: auto;
+			align-items: center;
+		}
+
+		.lu-select-display-wrapper {
+			overflow: hidden;
+			white-space: normal;
+			margin-bottom: calc(var(--spacings-smallest) * -1);
+		}
+	}
+
+	// Outlined
+	:host-context(.textfield.mod-outlined) {
+		&.textfield-input {
+			&::after {
+				color: var(--palettes-grey-600);
```

Comment:
You should use ths component specifics vars instead : `--components-field-label-color` ..